README.md

```diff
@@ -84,3 +84,25 @@ console.log(electron)
 // spawn electron
 var child = proc.spawn(electron)
 ```
+
+## ELECTRON_ENV environment variable
+
+The `electron` CLI adds an `ELECTRON_ENV` environment variable with the default value of `'development'`. The variable is inherited if set in a parent process.
+
+An Electron app can check the value of the environment variable and do different things based on the value. `ELECTRON_ENV` is used (by convention) to state whether a particular environment is a production or a development environment. A common use-case is running additional debugging or logging code if running in a development environment.
+
+You can check the variable like this:
+
+```js
+var environment = process.env.ELECTRON_ENV
+```
+
+Alternatively you can use [electron-is-dev](https://github.com/sindresorhus/electron-is-dev) module.
```

Comment:
Wouldn't this environment variable kind of render `electron-is-dev` unnecessary for most users?